### PR TITLE
feat: add avatar for lucy

### PR DESCRIPTION
**Pull request by LucyLu8L**

### CODE REVIEW

File: components/TeamSection.vue [hunk 1]
```diff
@@ -139,6 +139,11 @@ const people = [
     role: "Engineering Intern",
     imageUrl: require(`~/assets/people/qiaosheng.webp`),
   },
+  {
+    name: "Lucy",
+    role: "Community Intern",
+    imageUrl: require(`~/assets/people/lucy.webp`),
+    }
```
Comment (ecmadao) [inline components/TeamSection.vue:146]: Could you please change the indent here?

**Review by ecmadao — CHANGES_REQUESTED** [review]

**Review by ecmadao — DISMISSED** [review]
LGTM

**Comment by LucyLu8L** [comment]
resolved conflict, please review again @ecmadao

**Review by ecmadao — APPROVED** [review]

**Comment by tianzhou** [comment]
Welcome onboard~

**Pull request merged**